### DISCUSSION TITLE
Dynamic Layer Height improvement

### DISCUSSION THREAD

**Discussion by ecospline** [sn4k3/UVtools#851] [Ideas]
I suggest to add a possibility to use Dynamic Layer Height that allows to set it for multiple ranges - for example I have a print with 500 layers and I want to speed up this print in a two ranges-  from layer 0 to 100 and from layer 300 to 500. Maybe it can be achivable by possibility to adding another "Layer range selectors", in witch every one would have the possibility to set its own settings. 
If it's not possible - than maybe it's  doable to extend funcionality of  existing Layer range selector by allowing to set option of "include" or "exclude" for selected range ? - it would give a possibility to excude the range from layer 101 to 299 (in my example ) from beeing processed by DLH

**Comment by sn4k3** [comment]
Having multiple ranges would affect all program and in this tool will be hard to implement.
Maybe I can allow the tool to run multiple times

**Comment by Hazoooka** [comment]
The quality of the surface finish will worsen with two or more layer heights.
You will always have a line on the border.
